src/Controller/CollectionsController.php

```diff
@@ -29,6 +29,7 @@ public function collections() {
           'bundle' => 'image',
           'status' => 1,
           'field_media_of' => $collection_id,
+          'field_media_use' => 20,
```

Comment:
is this id guaranteed?

Reply:
This ID points to the `Thumbnail Image` taxonomy term, which as far as I know is just part of the system i.e. not likely to be changed by a system admin. Good point though, I'll ask someone to verify